app/models/manageiq/providers/lenovo/physical_infra_manager/event_catcher/stream.rb

```diff
@@ -54,7 +69,7 @@ def create_event_connection(ems)
                 :port => ems.endpoints.first.port)
   end
 
-  def get_last_cnn_from_events(ems_id)
-    EventStream.where(:ems_id => ems_id).maximum('ems_ref') || 1
+  def bigger_ems_ref(ems_id)
```

Comment:
I think `last_event_ems_ref` or something would be a better name.

Reply:
@agrare  It's fine, I changed this name to `last_event_ems_ref`, thanks :+1: